<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="7" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-20T11:14:24 MST" name="Suite" finished-at="2024-02-20T11:15:22 MST" duration-ms="57816">
    <groups>
    </groups>
    <test started-at="2024-02-20T11:14:24 MST" name="Test" finished-at="2024-02-20T11:15:22 MST" duration-ms="57816">
      <class name="tests.LoginTest">
        <test-method signature="login()[pri:0, instance:tests.LoginTest@7e5c856f]" started-at="2024-02-20T11:14:57 MST" name="login" finished-at="2024-02-20T11:15:22 MST" duration-ms="24579" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [395.70] but found [378]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [395.70] but found [378]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.pageObjectClasses.LoginDetails.comparePrice(LoginDetails.java:127)
at tests.LoginTest.login(LoginTest.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
      </class> <!-- tests.LoginTest -->
      <class name="tests.SearchFlightTest">
        <test-method is-config="true" signature="intializeBrowser()[pri:0, instance:tests.SearchFlightTest@5b367418]" started-at="2024-02-20T11:14:24 MST" name="intializeBrowser" finished-at="2024-02-20T11:14:29 MST" duration-ms="4113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intializeBrowser -->
        <test-method signature="searchForFlight()[pri:0, instance:tests.SearchFlightTest@5b367418]" started-at="2024-02-20T11:14:29 MST" name="searchForFlight" finished-at="2024-02-20T11:14:40 MST" duration-ms="11370" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchForFlight -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.SearchFlightTest@5b367418]" started-at="2024-02-20T11:15:22 MST" name="tearDown" finished-at="2024-02-20T11:15:22 MST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.SearchFlightTest -->
      <class name="tests.BookFlightTest">
        <test-method signature="BookFlight()[pri:0, instance:tests.BookFlightTest@12d2ce03]" started-at="2024-02-20T11:14:40 MST" name="BookFlight" finished-at="2024-02-20T11:14:57 MST" duration-ms="17368" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BookFlight -->
      </class> <!-- tests.BookFlightTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
